sbws/lib/v3bwfile.py

```diff
@@ -44,74 +46,82 @@
-    # because of network failures or it was
-    # not found a suitable helper relay
-    'recent_measurements_excluded_error_count',
-    # Number of relays that have successful measurements but the measurements
-    # were not away from each other in X time (by default 1 day).
-    'recent_measurements_excluded_near_count',
-    # Number of relays that have successful measurements and they are away from
-    # each other but they are not X time recent.
-    # By default this is 5 days, which is the same time the older
-    # the measurements can be by default.
-    'recent_measurements_excluded_old_count',
-    # Number of relays that have successful measurements and they are away from
-    # each other and recent
-    # but the number of measurements are less than X (by default 2).
-    'recent_measurements_excluded_few_count',
-]
-# Added in #29591
-# NOTE: recent_consensus_count, recent_priority_list_count,
-# recent_measurement_attempt_count and recent_priority_relay_count
-# are not reset when the scanner is stop.
-# They will accumulate the values since the scanner was ever started.
-BW_HEADER_KEYVALUES_MONITOR = [
-    # 1.1 header: the number of different consensuses, that sbws has seen,
-    # since the last 5 days
-    'recent_consensus_count',
-    # 2.4 Number of times a priority list has been created
-    'recent_priority_list_count',
-    # 2.5 Number of relays that there were in a priority list
-    # [50, number of relays in the network * 0.05]
-    'recent_priority_relay_count',
-    # 3.6 header: the number of times that sbws has tried to measure any relay,
-    # since the last 5 days
-    # This would be the number of times a relays were in a priority list
-    'recent_measurement_attempt_count',
-    # 3.7 header: the number of times that sbws has tried to measure any relay,
-    # since the last 5 days, but it didn't work
-    # This should be the number of attempts - number of ResultSuccess -
-    # something else we don't know yet
-    # So far is the number of ResultError
-    'recent_measurement_failure_count',
-    # The time it took to report about half of the network.
-    'time_to_report_half_network',
-] + BW_HEADER_KEYVALUES_RECENT_MEASUREMENTS_EXCLUDED
-BANDWIDTH_HEADER_KEY_VALUES_INIT = \
-    ['earliest_bandwidth', 'generator_started',
-     'scanner_country', 'destinations_countries']\
-    + STATS_KEYVALUES \
-    + BW_HEADER_KEYVALUES_MONITOR
-
-KEYVALUES_INT = STATS_KEYVALUES + BW_HEADER_KEYVALUES_MONITOR
-# List of all unordered KeyValues currently being used to generate the file
-UNORDERED_KEYVALUES = EXTRA_ARG_KEYVALUES + STATS_KEYVALUES + \
-                      ['latest_bandwidth'] + \
-                      BW_HEADER_KEYVALUES_MONITOR
-# List of all the KeyValues currently being used to generate the file
-ALL_KEYVALUES = ['version'] + UNORDERED_KEYVALUES
+# # KeyValues that count the number of relays that are in the bandwidth file,
```

Comment:
Why do we keep all this commented code around?

Reply:
> Why do we keep all this commented code around?

https://github.com/torproject/sbws/pull/357/files/8a5efb16a8bf25656d1fb515ac43202542f1dcec..ebbb9ffcf3ee28d48c68b7ef45463ed5f534e2ff#diff-0c88fa36ad3aabea4aa3c57e63f54db8R76:

```
# The commented code is kept so that
# it can be known what each Key means.
```

But certainly, i can just link to the specification where they are explained. And anyway those comments should now be in stem. Sounds good?

Reply:
Removed commented code

Reply:
ACK!